pkg/cvo/sync_worker.go

```diff
@@ -393,8 +557,19 @@ type statusWrapper struct {
 	previousStatus *SyncWorkerStatus
 }
 
+func (w *statusWrapper) ValidPayloadStatus(release configv1.Release) bool {
+	return w.previousStatus.loadPayloadStatus.Release.Image == release.Image
```

Comment:
I expect we want to use `return equalDigest(w.previousStatus.loadPayloadStatus.Release.Image, release.Image)` here, to avoid invalidating our status if someone moves `registryA@sha256:123...` to `registryB@sha256:123...`, like we sometimes see in QE.

Reply:
https://github.com/openshift/cluster-version-operator/pull/745